src/annotate.ts

```diff
@@ -7,10 +7,21 @@ import * as fs from 'fs';
 const schema: any = JSON.parse(fs.readFileSync('./schemas/annotate-is-valid.json', 'utf8'));
 const ajv = new Ajv({ allErrors: true, verbose: true });
 
-export default async function annotate({payload, _ajv = ajv, _schema = schema}: { payload: { field: string, term: string }, _ajv: typeof ajv, _schema: typeof schema}) {
+function confidenceScoreF(string: string) {
+  const obj = {
+    'LOW': 0,
+    'MEDIUM': 0,
+    'GOOD': 0,
+    'HIGH': 0
+  };
+  obj[string] = 1;
```

Comment:
I don't know if you really want to do this, but if so, do object assignation instead:
```ts
return {...obj, [key]: 1}
```